packages/admin/src/components/Scenario/CardListContainer/index.tsx

```diff
@@ -0,0 +1,45 @@
+import NoticeCardList from "@admin/components/Scenario/NoticeCardList";
+import { GroupContext } from "@admin/utils/groupContext";
+import { ScrapperConfig, ScrapperContext } from "@admin/utils/scrapperContext";
+import { StatusContext } from "@admin/utils/statusContext";
+import {
+  noticeScenariosMocks,
+  studentRestaurantScenariosMocks,
+  domitoryRestaurantScenariosMocks,
+  colleageScheduleMocks,
+} from "@admin/__mocks__";
+import { useContext } from "react";
+import CardList from "../CardList/CardList";
+import getStyle from "./style";
+
+export default function CardListContainer() {
+  const { group } = useContext(GroupContext);
+  const { scrapper } = useContext(ScrapperContext);
+  const { status } = useContext(StatusContext);
+  const { notice, studentRestaurant, domitoryRestaurant } = ScrapperConfig;
+  const style = getStyle();
+
+  let isNoticeScrapper = false;
+  let data;
+
+  switch (scrapper) {
+    case notice:
+      isNoticeScrapper = true;
+      data = noticeScenariosMocks;
+      break;
+    case studentRestaurant:
+      data = studentRestaurantScenariosMocks;
+      break;
+    case domitoryRestaurant:
+      data = domitoryRestaurantScenariosMocks;
+      break;
+    default:
+      data = colleageScheduleMocks;
+  }
+
+  return isNoticeScrapper ? (
+    <NoticeCardList style={style} group={group} data={data} status={status} />
+  ) : (
+    <CardList style={style} status={status} data={data} />
+  );
```

Comment:
제스가 말씀해주셨는데 ㅎ 컴포넌트도 이건 짧지만 길어지는 경우에는 삼항연산자로 알아보기 힘들더라구요!
이렇게 하시는건 어떨까요?
```suggestion
  if (isNoticeScrapper) return (
    <NoticeCardList {...{style,group,data,status}} />
  )
  return <CardList {...{style,status,data}} />;
```

Reply:
아하아하 확실히 삼항연산자보단 if 블록이 명확한 느낌이네요 더불어서 전개 연산자로 프롭들을 푸는 것도 깔끔한 것 같습니다!